qemu/tests/cfg/qmp_event_notification.cfg

```diff
@@ -91,3 +91,11 @@
                 - qmp_none:
                     extra_params += " -watchdog i6300esb -watchdog-action none "
                     action_check = "none"
+        - qmp_guest_panicked:
+            no Windows
```

Comment:
Does windows not support this test?

Reply:
@luckyh  first, no care about windows and linux from qmp event part, second, the method of  windows is verify different with linux, also need mannual settings for windows